src/cf-buttons/src/atoms/buttons.less

```diff
@@ -169,4 +170,14 @@
       padding-bottom: unit( @btn-v-padding-modifier-ie * 15px / @btn__super-font-size, em );
     }
   }
+
+  //
+  // Full width button on x-small screens
+  //
+  &__full-on-xsmall {
```

Comment:
I feel like consistency would be good. It would also prevent you from having to remember if you need a hyphen between `x` and `small` or `large`.